staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -68,6 +68,9 @@ var (
 		# Begin streaming the logs of the ruby container in pod web-1
 		kubectl logs -f -c ruby web-1
 
+		# Wait for atleast one container to be ready before streaming the logs
+		kubectl logs -f web-1 -w
```

Comment:
@ankritisachan so here let's look at `-c ruby` above, we should allow that firstly. Namely wait for the specified container and stream when it is ready.

second, if it is not specified, we should wait for ALL containers to be ready and not just one of them. 

(please think about the least surprise and best consistency and you will see why i am asking for this)